lib/fizzy/execute.cpp

```diff
@@ -376,6 +455,7 @@ execution_result execute(Instance& instance, FuncIdx func_idx, std::vector<uint6
     assert(code_idx < instance.module.codesec.size());
 
     const auto& code = instance.module.codesec[code_idx];
+    auto& memory = *instance.memory;
```

Comment:
How about making a commit/PR for this refactoring?

Reply:
Extracted it to a separate first commit here.